style/component/input.css

```diff
@@ -6,7 +6,7 @@
 .input:focus{
       outline: none;
       border-color: var(--primary-color);
-      box-shadow: 0 0 0 0.3rem var(--primary--shadow-color);
+      box-shadow: 0 0 0 0.3rem var(--primary-shadow-color);
```

Comment:
Please refrain from making changes out of context.

Reply:
Will keep it in mind in my next pr request. Thanks for the suggestion.